pkg/solana/config/toml.go

```diff
@@ -112,6 +112,7 @@ type TOMLConfig struct {
 	// Do not access directly, use [IsEnabled]
 	Enabled *bool
 	Chain
+	MultiNode
 	Nodes Nodes
```

Comment:
i think you're also missing handling multinode config in the `(c *TOMLConfig) SetFrom(f *TOMLConfig)` func?

Reply:
Yeah leaving the rest of the TOML configuration for another PR including the default values